client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -204,9 +209,8 @@ export const PrescriptionLineEditForm: React.FC<
   };
 
   const handlePrescribedQuantityChange = (inputPrescribedQuantity?: number) => {
-    if (inputPrescribedQuantity == null) return;
-    setPrescribedQuantity(inputPrescribedQuantity);
-    handleIssueQuantityChange(inputPrescribedQuantity, 'prescribed');
+    setPrescribedQuantity(inputPrescribedQuantity ?? 0);
+    handleIssueQuantityChange(inputPrescribedQuantity ?? 0, 'prescribed');
```

Comment:
Are we defaulting null / undefined to 0?

Reply:
I just copied what was going on for issue quan, bc that one works as expected 🤷

Reply:
Possibly should be nulled out, but I didn't know if that would have further type implications, wanted to move quick 👀